cmd/feed-dns/main.go

```diff
@@ -104,4 +116,14 @@ func validateConfig() {
 		log.Error("Must supply r53-hosted-zone")
 		os.Exit(-1)
 	}
+
+	if (internalCNAME != "" || externalCNAME != "") && elbLabelValue != "" {
```

Comment:
Probably want checks on command line arguments before you start using them, so these if blocks should move to the top of `main()`